variables.tf

```diff
@@ -137,3 +137,8 @@ variable "repository_credentials_kms_key" {
   description = "key id, key ARN, alias name or alias ARN of the key that encrypted the repository credentials"
   type        = string
 }
+
+variable "service_registry_arn" {
+  default = null
```

Comment:
The default value for a string can just be `""`.

Reply:
I think that null is more correct here.

Reply:
I could be wrong...